ui/src/components/landingpage/LandingPage.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+import Header from './Header'
```

Comment:
Path here will be updated as part of my pr so you can remove any reference to header from this version. 
You can see it on [this line](https://github.com/aerogear/mobile-security-service/blob/2a1c68be833d7cf7fa238e9690df83a238b708cc/ui/src/components/landingpage/LandingPage.js#L2) which is added as part of [this pr ](https://github.com/aerogear/mobile-security-service/pull/44/files)